Gruntfile.js

```diff
@@ -81,6 +81,11 @@ module.exports = function(grunt) {
 			tasks: {
 				src: ['polyfills/**/update.task.js'],
 			}
+		},
+		"deployvcl": {
+			dryrun: { options: {serviceId: "2l7QhcR5rkdg691tcdRTD7", dryRun: true} },
```

Comment:
Separate vars for each target, or do we expect the env to manage the right token?

So ie. `FASTLY_SERVICE_ID` or `FASTLY_SERVICE_ID_[QA|PROD]`?